ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -2,33 +2,36 @@
 import os
 import uuid
 
+from dcos_installer import backend
+
 
 class InstallerAnalytics():
-    def __init__(self, config):
+    def __init__(self):
         self.uuid = self.new_uuid()
-        self.bootstrap_version = os.getenv("BOOTSTRAP_ID")
-        self.customer_key = config["customer_key"] if 'customer_key' in config else ""
+        self.bootstrap_version = os.getenv("BOOTSTRAP_ID", "")
```

Comment:
this should hard fail if it isn't set / that is our internal code being inconsistent if it isn't set. A set bootstrap_id is mandatory